v2/sourcedb-to-spanner/src/test/resources/ReservedKeywordsIT/postgresql-schema.sql

```diff
@@ -0,0 +1,142 @@
+CREATE TABLE reserved_keywords (
+  id INT8 PRIMARY KEY,
```

Comment:
is it possible to have a reserved keyword in the primary key ?

i would suspect that most queries would use it in the where clause and it might break. Can you add a table with that test case?

Reply:
Done

Reply:
A note: As discussed over chat, we will need columns not named `id` as the first primary key and this might need a bit of deeper dive in the current  IT code to allow such a possibility.
Will take it up in MySQL side of the changes.